multimedia/whispercpp/Makefile

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2024 Dengfeng Liu <liudf0716@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=whispercpp
+PKG_VERSION:=1.5.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```

Comment:
Nope.

Reply:
sorry, can u give more information?

Reply:
@BKPepe I thought I had used the latest tagged version. What else do I need to do? I'm looking forward to your detailed explanation. Thank you.

Reply:
The source is going to be named: v1.5.4.tar.gz, which will be downloaded and with the same name, it will be put into the dl folder of your buildir. So, you would not know to which project it belongs. What if some file other will overwrite it? 

It needs to be downloaded as projectname-version.

This can be done by e.g.:

PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://codeload.github.com/ggerganov/whisper.cpp/tar.gz/v$(PKG_VERSION)?

Which can be tested by e.g.
https://codeload.github.com/ggerganov/whisper.cpp/tar.gz/v1.5.4?

Reply:
@BKPepe Thank you for your detailed explanation.

Reply:
@BKPepe Hey, I've updated the code based on your corrections. Are there any further changes needed?